demo/gdi/nuklear_gdi.h

```diff
@@ -408,6 +409,61 @@ nk_gdi_stroke_circle(HDC dc, short x, short y, unsigned short w,
     }
 }
 
+static void
+nk_gdi_stroke_arc(HDC dc, short cx, short cy, unsigned short r, float amin, float adelta, unsigned short line_thickness, struct nk_color col)
+{
+	COLORREF color = convert_color(col);
+
+	/* setup pen */
+    HPEN pen = NULL;
+    if (line_thickness == 1)
+        SetDCPenColor(dc, color);
+    else
+    {
+    	DWORD pen_style = PS_SOLID | PS_ENDCAP_FLAT | PS_GEOMETRIC; /* without that endcap round caps are used which looks really weird for thick arcs */
+
+    	LOGBRUSH brush;
+		brush.lbStyle = BS_SOLID;
+		brush.lbColor = color;
+		brush.lbHatch = 0;
+
+    	pen = ExtCreatePen(pen_style, line_thickness, &brush, 0, NULL);
```

Comment:
There seems to be some inconsistent spacing in this file generally. Idk if it's a mix of tabs and spaces.

Also, I wounder if there is a way we can reuse this `pen` or something to avoid allocating and freeing it on each call to this function (I assume `ExtCreatePen` allocates memory, since it has to be deleted). I don't think this is a showstopper for getting this merged, but it would be an improvement IMHO.

Reply:
Many functions in the GDI backend seem to do this but with CreatePen instead of ExtCreatePen; basically a simpler pen constructor that requires a deletion just like ExtCreatePen.
It might be a good idea to optimize this in an extra pull request if possible since it affects more than just the new functions.